packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -245,10 +245,12 @@ class ToolbarOptions {
     this.cut = false,
     this.paste = false,
     this.selectAll = false,
+    this.actions = const <TextSelectionAction>[],
```

Comment:
There are two places where the docs need to be updated above here.

```
All options default to false if they are not explicitly set.
```
=>
```
All boolean options default to false if they are not explicitly set.
```

---

```
It includes several options: cut, copy, paste, and select all.
```
=>
```
It includes several actions by default: cut, copy, paste, and select all.
```